samples/drivers/CAN/prj.conf.mcp2515

```diff
@@ -0,0 +1,13 @@
+# CAN + MCP2515 config
+CONFIG_SPI=y
+CONFIG_CAN=y
+CONFIG_CAN_INIT_PRIORITY=80
+CONFIG_CAN_1=y
+CONFIG_CAN_STM32=n
+CONFIG_CAN_MCP2515=y
+CONFIG_GPIO_LED_DEV="GPIOB"
```

Comment:
already discussed earlier but noted again here as a to do for later:
use `led0` and `sw0` from the board's device tree.